httpmw/logging/logging.go

```diff
@@ -0,0 +1,71 @@
+package logging
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/gorilla/mux"
+)
+
+type Logger interface {
+	Info(msg string, keysAndValues ...interface{})
+	Error(msg string, keysAndValues ...interface{})
+}
+
+// more info about gorilla/mux mw here:
+// https://github.com/gorilla/mux/tree/db9d1d0073d27a0a2d9a8c1bc52aa0af4374d265?tab=readme-ov-file#middleware
```

Comment:
This is a better link: https://pkg.go.dev/net/http#Handler

Reply:
Oh yes, that is good for info about the http handler type. Added.